src/extension/wps/src/main/java/au/org/emii/wps/GoGoDuckProcess.java

```diff
@@ -75,20 +68,24 @@ public FileRawData execute(
                 throw new ProcessException("fileLimit set to 0 or below, job will not run");
             }
 
-            final File outputFile = getResourceManager().getOutputResource(
+            if (filter != null && !filter.isEmpty()) {
```

Comment:
So this should be a comma separated list of filters. Make an array out of it, and pass it to GGD.